api/proto/config/edgeview.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2021 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package  = "github.com/lf-edge/eve/api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+message EdgeViewConfig {
+    // JWT token for signed info, it contains the dispatcher
+    // endpoint IP:Port, device UUID and expiration time
+    string  token = 1;
+    // dispatcher certificate(s) if it's not well-known CA signed
+    repeated bytes disp_cert_pem = 2;
+    // policy for device access through edge-view
+    DevEdgeViewPolicy dev_policy = 3;
+    // policy for apps through edge-view
+    AppEdgeViewPolicy app_policy = 4;
+}
+
+message DevEdgeViewPolicy {
+    // device side of edge-view access is alllowed or not
+    bool allow_dev = 1;
+    // allow ssh into device or not, ssh key still need to be
+    // installed separately even the edge-view allows it
+    bool ssh = 2;
+    // allow to issue reboot device or not
+    bool reboot = 3;
+    // allow to launch VNC into application console or not; the
+    // app still need to enable VNC access even it is allowed here
+    bool app_console = 4;
+}
+
+message AppEdgeViewPolicy {
+    // app side of edge-view access is allowed or not
+    bool allow_app = 1;
+    // ssh through port 22 into app is allowed or not
+    bool ssh = 2;
+    // the https proxy is allowed or not
+    bool proxy = 3;
+    // only the defined set of tcp ports allowed
+    repeated int32 only_ports = 4;
+    // the ports listed are not allowed
+    repeated int32 disallow_ports = 5;
+    // only the listed IP-Addresses or Domain Names are allowed
```

Comment:
Hard to understand, if nothing defined here, is it mean that nothing allowed?

Reply:
No, if nothing defined for 'only_ports' and nothing defined for 'disallow_ports', then everything is good. is allowed. if only_ports is defined, then I guess there is no need to add 'disallow_ports'; but if 'disallow_ports' are defined, then normally one will not put only_ports. I guess the 'only_ports' if people knows exactly what to allow, like we do today in physical port-mapping. but if they are not sure, maybe use 'disallow_ports' makes sense.

Reply:
the implementation part: https://github.com/lf-edge/eve/pull/2398/files#diff-d6035f2b614254f0ea25e043313856bb6d844dc73f7f4f5b8c8b56dfedadefeeR160

Reply:
I'd suggest we reuse the ACL API definitions we already have if/when we need to specify ports etc.